include/vrapp/VolumeVisualizationApp.h

```diff
@@ -60,59 +57,50 @@ class Texture;
 class UIView;
 class VRVolumeApp;
 
-class VolumeVisualizationApp : public VRApp {
+class VolumeVisualizationApp : public VRApp
+{
 public:
-
   /** The constructor passes argc, argv, and a MinVR config file on to VRApp.
    */
-  VolumeVisualizationApp(int argc, char** argv);
+  VolumeVisualizationApp(int argc, char **argv);
   virtual ~VolumeVisualizationApp();
 
-
-
   /** USER INTERFACE CALLBACKS **/
 
-  virtual void onCursorMove(const VRCursorEvent& state);
+  virtual void onCursorMove(const VRCursorEvent &state);
 
-  virtual void onAnalogChange(const VRAnalogEvent& state);
+  virtual void onAnalogChange(const VRAnalogEvent &state);
```

Comment:
should these all be snake case?

Reply:
This are the overwritten functions from the MinVR api.